### PR TITLE
Refactor test_generate_hashes_all_platforms to use local packages.

### LINKED ISSUE #1003: Refactor test_generate_hashes_all_platforms
Hm, this test checks that `pypi_repository.get_hashes()` returns hashes for all platforms (see wheels list for [cffi](https://pypi.org/project/cffi/#files). So we can resolve this by add another local package and generate weels for a few platforms.

_Originally posted by @atugushev in https://github.com/jazzband/pip-tools/pull/998#discussion_r349934356_

---

Add new wheels for `small-fake-multi-arch` package with version `0.1` for platforms "manylinux1_x86_64", "manylinux1_i686" and "win32", and move the wheels to `tests/test_data/minimal_wheels`.

Hints:

```console
$ cat setup.py
from setuptools import setup
setup(
    name="small-fake-multi-arch",
    version=0.1
)

$ python setup.py bdist_wheel --universal --plat-name manylinux1_x86_64
$ python setup.py bdist_wheel --universal --plat-name manylinux1_i686
$ python setup.py bdist_wheel --universal --plat-name win32
```

### PR DESCRIPTION
**Pull request by hramezani**
Fixes #1003.

**Changelog-friendly one-liner**: Add test_generate_hashes_all_platforms_local_package test.

##### Contributor checklist

- [x] Provided the tests for the changes.
- [x] Gave a clear one-line description in the PR (that the maintainers can add to CHANGELOG.md on release).
- [ ] Assign the PR to an existing or new milestone for the target version (following [Semantic Versioning](https://blog.versioneye.com/2014/01/16/semantic-versioning/)).

### CODE REVIEW

**Comment by hramezani** [comment]
> Great work! Wouldn't it make more sense to replace `test_generate_hashes_all_platforms` by `test_generate_hashes_all_platforms_local_package`?

Yes, it would.
I replaced `test_generate_hashes_all_platforms` with `test_generate_hashes_all_platforms_local_package`.

File: tests/test_repository_pypi.py [hunk 1]
```diff
@@ -6,53 +6,16 @@
 from piptools.repositories.pypi import open_local_or_remote_file
 
 
-@pytest.mark.network
-def test_generate_hashes_all_platforms(from_line, pypi_repository):
+def test_generate_hashes_all_platforms_local_package(
```
Comment (atugushev) [inline tests/test_repository_pypi.py]:
```suggestion
def test_generate_hashes_all_platforms(
```
I think we could keep the name of the test.

Comment (hramezani) [reply tests/test_repository_pypi.py]: Done.

**Comment by atugushev** [comment]
@hramezani, thanks as always! 🙏

**Pull request merged**